wasmaudioworklet/audioworkletpolyfill.js

```diff
@@ -6,12 +6,21 @@
  */
 
 if(typeof AudioContext !== 'function') {
-    window.AudioContext = window.webkitAudioContext;
+    /** @type {any} */
+    const win = window
+    window.AudioContext = win.webkitAudioContext;
 }
 
 if(typeof AudioWorkletNode !== 'function') {
     console.log('No audioworklet support - using polyfill');
-    window.AudioWorkletNode = function(context, processorName) {
+
+    window.AudioWorkletNode = /** @type {AudioWorkletNode} */(AudioWorkletNodePolyfill)
+
+    /** 
+     * @param {any} context
+     * @param {any} processorName
```

Comment:
As a quick escape hatch, just give things type `any`. Then it is as if we wrote plain JS anyways.

I'm just getting it set up, I'll update the types a little later...